src/client/sandbox/node/node-list-wrapper.js

```diff
@@ -15,27 +15,15 @@ export default class NodeListWrapper {
             }
         });
 
-        Object.defineProperty(this, '_nodeList', {
-            value: nodeList
-        });
-
-        Object.defineProperty(this, '_filterListFn', {
-            value: filterListFn
-        });
-
-        Object.defineProperty(this, '_filterItemFn', {
-            value: filterItemFn
-        });
+        Object.defineProperty(this, '_nodeList', { value: nodeList });
 
-        Object.defineProperty(this, '_filteredNodeList', {
-            get: () => this._filterListFn(this._nodeList, this._filterItemFn)
-        });
+        Object.defineProperty(this, '_filterListFn', { value: filterListFn });
 
+        Object.defineProperty(this, '_filterItemFn', { value: filterItemFn });
```

Comment:
I think `_filterItemFn` is redundant here. We can do the following when the wrapper is created:
```js
return new NodeListWrapper({
    nodeList,
    filterListFn: items => ShadowUI._filterList(items, ShadowUI._filterElement),
    isLiveCollection
});
```